interview-questions.md

```diff
@@ -7,62 +7,69 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 INSTRUCTOR EXAMPLE: What is a conditional statement?
 
-  Your answer:
+  Your answer: A conditional statment checks data against a condition and returns a specific output depending on the which condition it meets- The final else statement provides a failsafe option should none of the other conditions be met. 
 
   Researched answer:
 
 
 
 1. What is git? What is the difference between git and Github?
 
-  Your answer:
+  Your answer: git is version control software. It allows for teams or individuals to work on projects together and track the changes made to their documents. Github is the cloud software that utilizes git technology. When it isn't functioning correctly it stands for, 'god damn idiot truck-load of shit' - I've forgotten what it stands for when it is working. Interestingly though, not becuase I'd rather call it by it's crude name- I've found it got considerably easier to use after a couple of attempts. I anticipate being quite comfortable with it soon. 
```

Comment:
Lol, impressive that you remember the funny acronym the maker had for git.  Great answer!